apps/nextjs-website/src/components/PreFooter.tsx

```diff
@@ -6,15 +6,23 @@ import { PreFooterProps } from '@react-components/types';
 const makePreFooterProps = ({
   storeButtons,
   background,
+  ctaButtons,
   ...rest
 }: PreFooterSection): PreFooterProps => ({
-  ...(background.data && { background: background.data.attributes.url }),
+  ...(background?.data && { background: background.data.attributes.url }),
   ...(storeButtons && {
     storeButtons: {
       ...(storeButtons.hrefGoogle && { hrefGoogle: storeButtons.hrefGoogle }),
       ...(storeButtons.hrefApple && { hrefApple: storeButtons.hrefApple }),
     },
   }),
+  ...(ctaButtons && {
+    ctaButtons: ctaButtons.map((button) => ({
+      text: button.text,
+      href: button.href,
+      variant: button.variant || 'contained',
+    })),
+  }),
```

Comment:
```suggestion
  ...(ctaButtons.length > 0 && { ctaButtons }),
```
No parsing necessary, also check the array's length, not its existence, since it's always defined.